### ISSUE TITLE
Pix2PixModel: G_L1 retains lambda_L1 multiplier

### ISSUE THREAD

**Issue by asy51** [junyanz/pytorch-CycleGAN-and-pix2pix#1492]
When I tune the lambda_L1 hyperparameters, the corresponding G_L1 loss retains the lambda_L1 multiplier. In the image below, the second row (G_L1 loss = 0.1121; lambda_L1 = 10) performs the best (as far as I can tell), but the losses cannot be compared directly.

![image](https://user-images.githubusercontent.com/90255040/195203830-3debd588-8946-419a-a954-a9438ed99aa0.png)

Defining the Pix2PixModel.get_current_losses() function as shown below may mitigate this problem, but incurs other issues (e.g. losses within a model cannot be compared directly anymore).

```
# Pix2PixModel
    def get_current_losses(self):
        """
        Cancels out lambda_L1 multiplier for comparing L1 losses across models with different lambda_L1 values
        """
        errors_ret = super().get_current_losses()
        errors_ret['G_L1'] /= self.opt.lambda_L1
        return errors_ret
```

**Comment by taesungp** [comment]
Right, so there is a tradeoff as you pointed out. Either you can compare among losses within the same model, or compare them across different lambdas. For simplicity, we chose the former. Perhaps you can modify `get_current_losses()` so that it returns both numbers (unnormalized and normalized by the lambda)?
